mappings/net/minecraft/class_2813.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_2813
```

Comment:
this might be `GlStateManager`

Reply:
Take a closer look. It's quite hard to tell. This is why it's important to differentiate between B3D and the original implementation. It may need entirely new mappings.

Reply:
If it stores the gl cap states then it’s functionally equivalent to GlStateManager

Reply:
We can figure that out later, I guess. For now, I think we can resolve this conversation.